### DISCUSSION TITLE
Add email notification when meeting is deleted

### DISCUSSION THREAD

**Discussion by mikec-ncsd** [code4recovery/12-step-meeting-list#816] [Ideas]
Currently TMSL sends an email notification when a meeting is added or changed. It would also be useful if a meeting deletion would also trigger an email notification. 
My Central Office manager uses TMSL email notifications to track requests implemented in our website.

**Comment by joshreisner** [comment]
we are tracking this issue here: https://github.com/code4recovery/12-step-meeting-list/issues/503

thank you for the upvote
